setuptools/tests/test_easy_install.py

```diff
@@ -1011,7 +1011,22 @@ def test_get_script_header_exe_with_spaces(self):
         actual = ei.ScriptWriter.get_header(
             '#!/usr/bin/python',
             executable='"' + self.exe_with_spaces + '"')
-        expected = '#!"%s"\n' % self.exe_with_spaces
```

Comment:
This test has been changed because the shebang line that is generated is invalid.